src/CommunityToolkit.Maui.Core/Handlers/CameraView/CameraManager.ios.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System.Runtime.InteropServices;
+using AVFoundation;
+using CommunityToolkit.Maui.Core.Primitives;
+
+namespace CommunityToolkit.Maui.Core.Handlers;
+
+public partial class CameraManager
+{
+	AVCaptureSession? captureSession;
+	AVCapturePhotoOutput? photoOutput;
+
+	internal Action? Loaded { get; set; }
+
+	// IN the future change the return type to be an alias
+	public UIView CreatePlatformView()
+	{
+		captureSession = new AVCaptureSession
+		{
+			SessionPreset = AVCaptureSession.PresetPhoto
+		};
+
+		photoOutput = new AVCapturePhotoOutput();
+
+		var previewView = new PreviewView();
+		previewView.Session = captureSession;
+
+		return previewView;
+	}
+
+	public void Dispose()
+	{
+		Dispose(true);
+		GC.SuppressFinalize(this);
+	}
+
+	protected virtual partial void PlatformConnect()
+	{
+		var device = AVCaptureDevice.GetDefaultDevice(AVMediaTypes.Video) ?? throw new InvalidOperationException();
+
+		ArgumentNullException.ThrowIfNull(captureSession);
+		ArgumentNullException.ThrowIfNull(photoOutput);
+
+		captureSession.AddInput(new AVCaptureDeviceInput(device, out var err));
+		captureSession.AddOutput(photoOutput);
+		captureSession.StartRunning();
+	}
+
+	protected virtual partial void PlatformDisconnect()
+	{
+	}
+
+	protected virtual partial void PlatformTakePicture()
+	{
+		ArgumentNullException.ThrowIfNull(photoOutput);
+
+		var wrapper = new AVCapturePhotoCaptureDelegateWrapper();
```

Comment:
Instead of creating a new delegate each time, how about we create a new `TaskCompletionSource` every time and keep one instance of the delegate?

Reply:
How would you handle concurrent flows / multiple tcs?

Reply:
I could also implement AVCapturePhotoCaptureDelegate on the `CameraManager` if you prefer that.

Reply:
> How would you handle concurrent flows / multiple tcs?

This code should run on UIThread, so I don't think we'll have concurrent flows... Or iOS allow the camera to be triggered from a background thread?

> I could also implement AVCapturePhotoCaptureDelegate on the CameraManager if you prefer that.

Need more time to think about it